rocAL/source/api/rocal_api_augmentation.cpp

```diff
@@ -2431,3 +2431,81 @@ RocalTensor rocalNormalDistribution(RocalContext p_context,
     }
     return output;
 }
+
+std::pair<RocalTensor, RocalTensor> ROCAL_API_CALL
+rocalNonSilentRegion(
+        RocalContext p_context,
+        RocalTensor p_input,
+        bool is_output,
+        float cutoff_db,
+        float reference_power,
+        int reset_interval,
+        int window_length) {
+    Tensor* output1 = nullptr;
+    Tensor* output2 = nullptr;
+    if ((p_context == nullptr) || (p_input == nullptr))
+        ERR("Invalid ROCAL context or invalid input tensor")
+    TensorList output_tensors;
```

Comment:
Why do we need to keep it in a tensor list if we are going to return a pair of output1 and output2 ?